lib/ToastContainer.js

```diff
@@ -219,7 +219,7 @@ class ToastContainer extends Component {
             top: 0,
             bottom: this._keyboardHeight
         };
-
+        const InnerView = this.props.view;
```

Comment:
```suggestion
        const InnerView = this.props.view;
```
My suggestion => Delete this line